samples/subsys/input/input_dump/boards/esp32_devkitc_wroom.overlay

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2023 Espressif Systems (Shanghai) Co., Ltd.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/dt-bindings/input/input-event-codes.h>
+#include <zephyr/dt-bindings/input/esp32-touch-sensor-input.h>
+
+/ {
+	touch_sensors: esp-touch-sensor0 {
+		compatible = "espressif,touch-sensor";
+		status = "okay";
+		debounce-interval-ms = <30>;
+		href-microvolt = <2700000>;
+		lref-microvolt = <500000>;
+		href-atten-microvolt = <1000000>;
+		filter-mode = <ESP32_TOUCH_FILTER_MODE_IIR_16>;
+		filter-debounce-cnt = <1>;
+		filter-noise-thr = <ESP32_TOUCH_FILTER_NOISE_THR_4_8TH>;
+		filter-jitter-step = <4>;
+		filter-smooth-level = <ESP32_TOUCH_FILTER_SMOOTH_MODE_IIR_2>;
```

Comment:
Can you please move those huge initializers in the `dtsi` instead of samples overlay?
In the simple `.overlay` I'd expect only a few necessary changes as `status="okay"` and channel selection.